hudi-common/src/main/java/org/apache/hudi/common/model/HoodieRecordLocation.java

```diff
@@ -32,6 +32,7 @@
 public class HoodieRecordLocation implements Serializable, KryoSerializable {
   public static final long INVALID_POSITION = -1L;
 
+  // Instant time of the base file where the record is stored
```

Comment:
This is not true.

Reply:
For Spark, when doing SIMPLE or BLOOM index, this instant time is assigned from the base file commit time.  Is there any other case that this does not represent the base file commit time (e.g., in Flink)?

Reply:
But yes, if it is used for new location, that contains the instant time of the transaction, not the base file instant time.

Reply:
Revised the docs to be clear.

Reply:
Removed to avoid confusion.